File: Plugins/Aspose_Tasks_Java_for_Ruby/lib/asposetasksjava/Tasks/rendertasksheetview.rb
module Asposetasksjava
  module RenderTaskSheetView
    def initialize()
        data_dir = File.dirname(File.dirname(File.dirname(File.dirname(__FILE__)))) + '/data/'
                
        # Instantiate project object
        project = Rjb::import('com.aspose.tasks.Project').new(data_dir + 'test_tasks.mpp')

        # Define the SaveOptions with required TimeScale settings as Days
    	options = Rjb::import('com.aspose.tasks.PdfSaveOptions').new
    	# Set the Presentation format to ResourceUsage
    	options.setPresentationFormat(Rjb::import('com.aspose.tasks.PresentationFormat').TaskSheet)
    	# Save the Project
    	project.save(data_dir + "taskSheet.pdf", options)

    	puts "Created task sheet view file."
    end
  end
end

File: Plugins/Aspose_Tasks_Java_for_PHP/src/aspose/tasks/WorkingWithTasks/RenderTaskSheetView.php
<?php
namespace Aspose\Tasks\WorkingWithTasks;

use com\aspose\tasks\Project as Project;
use com\aspose\tasks\PdfSaveOptions as PdfSaveOptions;
use com\aspose\tasks\PresentationFormat as PresentationFormat;

class RenderTaskSheetView{

    public static function run($dataDir=null){

        # Instantiate project object
        $project = new Project($dataDir . 'test_tasks.mpp');

        # Define the SaveOptions with required TimeScale settings as Days
    	$options = new PdfSaveOptions();

        # Set the Presentation format to ResourceUsage
        $presentationFormat=new PresentationFormat();
        $options->setPresentationFormat($presentationFormat->TaskSheet);
    	# Save the Project
    	$project->save($dataDir . "taskSheet.pdf", $options);
    	print "Created task sheet view file.".PHP_EOL;
    }
}
?>